### PR TITLE
Some Windows tweaks

### PR DESCRIPTION
**Pull request by quark-zju**
Some random tweaks I made to make wezterm work reasonably on my Windows machine.

### CODE REVIEW

File: window/src/os/windows/window.rs [hunk 1]
```diff
@@ -779,12 +781,30 @@ unsafe fn mouse_move(hwnd: HWND, _msg: UINT, wparam: WPARAM, lparam: LPARAM) ->
     }
 }
 
+lazy_static! {
+    static ref WHEEL_SCROLL_LINES: i16 = read_scroll_speed("WheelScrollLines").unwrap_or(3);
+    static ref WHEEL_SCROLL_CHARS: i16 = read_scroll_speed("WheelScrollLines").unwrap_or(3);
```
Comment (quark-zju) [inline window/src/os/windows/window.rs]: This should be `WheelScrollChars` if you can fix in flight. Otherwise I can update the PR.

Comment (wez) [reply window/src/os/windows/window.rs]: I've fixed this locally; will push shortly!

File: window/src/os/windows/window.rs [hunk 2]
```diff
@@ -42,6 +42,9 @@ pub(crate) struct WindowInner {
     bitmap: RefCell<GdiBitmap>,
     #[cfg(feature = "opengl")]
     gl_state: Option<Rc<glium::backend::Context>>,
+    /// Fraction of mouse scroll
+    hscroll_reminder: i16,
```
Comment (wez) [inline window/src/os/windows/window.rs]: I've renamed these to "remainder" locally :)

**Comment by wez** [comment]
Looks great; pushed as 70e9c7cb16c6593166adfc3267ff0ca929344d33
Thanks!

**Pull request closed without merging**